docs/references/react/use-organization-list.mdx

```diff
@@ -1,3 +1,267 @@
 ---
-sanity_slug: /docs/reference/clerk-react/useorganizationlist
+title: useOrganizationList()
+description: Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
 ---
+
+import { Callout } from "nextra-theme-docs";
+import { Tables } from "@/components/Table";
+import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+
+# `useOrganizationList()`
+
+Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
+
+`useOrganizationList` enables developers to fetch the memberships, invitations or suggestsions for an active user.
+
+`Note: these resources are paginated`
+
+The hook requires developers to opt-in by specifying which resource they need to fetch and paginate through. It is ideal for infinite lists or tables
+
+## Usage
+### Infinite list of joined organizations
+<CodeBlockTabs options={['Next.js', 'React']}>
```

Comment:
We should have app router and pages router examples for Next.js,
so we can use <Tabs> for Next.js versus React examples, 
and for the Next.js tab, add <CodeBlockTabs options={["App router", "Pages router"}]

Reply:
tbh, i don't see why this would be use full. Code will be identical and it is already identical between the 2 tabs.

if we can remove them completely it be nice. I'm not opinionated, so if what u mentioned is a pattern we want to have across our docs, then do it.

Reminder that this has to do with hooks. Same code should run in any react related framework

Reply:
very good point !! ignore my comment